package.json

```diff
@@ -64,69 +64,69 @@
     "watch": "rollup -w -c"
   },
   "devDependencies": {
-    "@octokit/rest": "^16.28.4",
-    "@types/cheerio": "^0.22.12",
+    "@octokit/rest": "^16.28.7",
+    "@types/cheerio": "^0.22.13",
     "@types/cosmiconfig": "^5.0.3",
-    "@types/express": "^4.17.0",
-    "@types/jest": "^24.0.15",
+    "@types/express": "^4.17.1",
+    "@types/jest": "^24.0.18",
     "@types/jest-plugin-context": "^2.9.2",
-    "@types/node": "^12.6.2",
+    "@types/node": "^12.7.2",
     "@types/pug": "^2.0.4",
-    "@types/puppeteer": "^1.12.4",
+    "@types/puppeteer": "^1.19.1",
```

Comment:
We have mismatched version between puppeteer-core and its type definition, but we have to use this version because `@types/puppeteer` for v1.17.x is not provided.